permission_validators/main/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+//! Main and default Iroha instruction validator.
+
+#![no_std]
+#![no_main]
+
+extern crate alloc;
+
+use iroha_wasm::validator::prelude::*;
+
+/// Validate any [`Instruction`](iroha_wasm::data_model::isi::Instruction).
+//
+// TODO: Not exhaustive list. Add more validators here.
+#[entrypoint(params = "[authority, instruction]")]
+pub fn validate(authority: <Account as Identifiable>::Id, instruction: Instruction) -> Verdict {
```

Comment:
> Is this scenario possible? Would this fail with the current architecture and should it fail?

1. It is possible
2. It might, need more tests
3. No.

Reply:
Yes, we need more tests, I've already created #3242